tools/integration_tests/implicitdir/write_test.go

```diff
@@ -35,3 +35,19 @@ func TestWriteAtEndOfFile(t *testing.T) {
 
 	compareFileContents(t, fileName, "line 1\nline 2\nline 3\n")
 }
+
+func TestWriteAtStartOfFile(t *testing.T) {
+	fileName := createTempFile()
+	f, err := os.OpenFile(fileName, os.O_WRONLY, 0600)
+	if err != nil {
+		t.Errorf("Open file for write at start: %v", err)
+	}
+
+	if _, err = f.WriteAt([]byte("line 4\n"), 0); err != nil {
+		t.Errorf("WriteString-Start: %v", err)
+	}
+
```

Comment:
in test cases, we should have the formatting like this
test initialisation
<new line>
test invocation
<new line>
assert statements.

f.close is part of test invocation only. so remove new line before f.close.